packages/libs/react-ui/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -53,9 +61,14 @@ export const Controlled: Story = {
     label: 'Label',
```

Comment:
Another improvement that could be made, which isn't directly part of this PR, is to add a description to this prop/argument in storybook as it's not directly clear where or how this label is used.